recipes/_alfrescoproperties-attributes.rb

```diff
@@ -68,6 +68,11 @@
 node.default['alfresco']['properties']['opencmis.server.host'] = node['alfresco']['public_hostname']
 node.default['alfresco']['properties']['opencmis.server.protocol'] = node['alfresco']['public_protocol']
 
+# Activiti
```

Comment:
I'd move this block in `recipes/_activiti-attributes.rb` ; this way, if activiti component is not enabled, these alfresco-global properties won't be added